package.json

```diff
@@ -11,7 +11,7 @@
     "url": "git://github.com/Hylozoic/hylo-node.git"
   },
   "engines": {
-    "node": "^14.17.6",
+    "node": "^16",
```

Comment:
This vagueness is on purpose in the spirit of this designator evidently in this context being the "minimum version" 🤷🏼‍♂️